integration-tests/billing/services/fixture-loader/adapters/BookshelfAdapter.js

```diff
@@ -43,6 +41,16 @@ class BookshelfAdapter {
       }
     }
   }
+
```

Comment:
This integration test file isn't being run and so I added the pick function as I know this is working. This file will soon be deleted as part of an issues ticket the team is working on.